src/nix/flake.cc

```diff
@@ -1132,7 +1132,8 @@ struct CmdFlakeShow : FlakeCommand, MixJSON
                     }
                 }
 
-                else {
+                // Ignore 'systems' and 'perSystem' attributes
+                else if (!(attrPathS[0] == "systems" || attrPathS[0] == "perSystem")) {
```

Comment:
If i do `attrPathS[0] != "systems" && attrPathS[0] != "perSystem"` it throws compilation error:
> src/nix/[flake.cc:1136](http://flake.cc:1136/):68: error: no match for ‘operator!=’ (operand types are ‘__gnu_cxx::__alloc_traits<std::allocator<nix::SymbolStr>, nix::SymbolStr>::value_type’ {aka ‘nix::SymbolStr’} and ‘const char [10]’)